staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -435,6 +436,11 @@ func AddFieldManagerFlagVar(cmd *cobra.Command, p *string, defaultFieldManager s
 	cmd.Flags().StringVar(p, "field-manager", defaultFieldManager, "Name of the manager used to track field ownership.")
 }
 
+// AddLabelFlagVar adds label flag to a command.
+func AddLabelFlagVar(cmd *cobra.Command, value *string) {
+	cmd.Flags().StringVarP(value, "label", "b", *value, i18n.T("Add labels as key=val pairs (e.g. -b label1=val1,label2=val2,label3=val3)."))
```

Comment:
You can use StringSlice here (or StringSliceVar), so for each combination of "key=value,key=value" it became a new item in the array